core/bin/external_node/src/config/mod.rs

```diff
@@ -227,10 +227,36 @@ impl RemoteENConfig {
             l2_legacy_shared_bridge_addr: Some(Address::repeat_byte(7)),
             l1_batch_commit_data_generator_mode: L1BatchCommitmentMode::Rollup,
             dummy_verifier: true,
+            l2_timestamp_asserter_addr: None,
         }
     }
 }
 
+async fn handle_rpc_response_with_fallback<T, F>(
+    rpc_call: F,
+    fallback: Option<T>,
+    context: String,
+) -> anyhow::Result<T>
+where
+    F: Future<Output = Result<T, ClientError>>,
+    T: Clone,
+{
+    match rpc_call.await {
+        Err(ClientError::Call(err))
+            if [
+                ErrorCode::MethodNotFound.code(),
+                // This what `Web3Error::NotImplemented` gets
+                // `casted` into in the `api` server.
+                ErrorCode::InternalError.code(),
+            ]
+            .contains(&(err.code())) =>
+        {
+            fallback.ok_or_else(|| anyhow::anyhow!("RPC method not found or internal error"))
```

Comment:
I don't understand how `fallback == None` works; as it's implemented currently, it's not a fallback at all, and a node executing this code will still break down here unless the main node is updated. Isn't `None` a valid value for the `timestamp_asserter_address`? If so, it looks like `fallback` should have type `T` and it should be returned here unconditionally.

Reply:
Good point - fixed